concurrency/3.0/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Jakarta Concurrency 3.0 (Under Development)"
+date: 2021-10-15
+summary: "Jakarta EE 10 Release"
+---
+Jakarta Concurrency provides a specification for using concurrency from application components without compromising container integrity while still preserving the Jakarta EE platform's fundamental benefits.
+
+# Plan
+Jakarta Concurrency 3.0 will target the Jakarta EE 10 platform release. 
+
+Jakarta Concurrency 3.0 will aim to contain.
+
+Features identified by the community on GitHub tagged with 3.0 see 3.0 Milestone (github.com)
+
+Bug Fixes as they arise during the release cycle
+
+Documentation clean up as identified during the release cycle
+
+Any updates required to meet the Java version requirements of Jakarta EE 10
+
+Any requirements identified by other specifications or the Jakarta EE 10 platform projects during the release cycle.
+
+* [Jakarta Concurrency 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.cu/releases/3.0.0)
```

Comment:
Specifications should only have a two digit major.minor version.  Your text for the link is good (3.0), but the actual release record should also be 3.0 (not 3.0.0).  Thanks!